### ISSUE TITLE
The test-item GET has a broken launchesLimit parameter

### ISSUE THREAD

**Issue by maximilian-mueller67** [reportportal/reportportal#1910]
**Describe the bug**
The launchesLimit parameter in use with isLatest parameter has unexpected behaviour.
Example: Get all the test items for the latest launch with a filter id
http://IP/api/v1/default_personal/item?page.size=300&page.page=1&filterId=FILTERID&isLatest=true&launchesLimit=1

The above request will return no items.
if i run it again but with isLatest=false then i will get some results.

if i set isLatest=true and use launchesLimit >= 2, then results will be returned. 

So basically the behaviour is if isLatest=true, then only return results if launchesLimit >= 2. if launchesLimit == 1, return no results. if launchesLimit == 0, return error
```
{
    "errorCode": 40016,
    "message": "Error in handled Request. Please, check specified parameters: 'Launches limit should be greater than 0'"
}
```

**Expected behavior**
if isLatest=true, then launchesLimit is unnecessary. Even if it should be included for completeness, then the launchesLimit=1 should return the same result as launchesLimit=100.
It even says it in the above error message that the launchesLimit should be greater than 0, so 1 needs to be valid.

**Versions:**
 - RP: Linux, Client: Windows, Linux
 - API Service: 5.7.3;Index Service: 5.7.3;Jobs Service: 5.7.3;Authorization Service: 5.7.3;Service UI: 5.7.3;

**Comment by alena-babrova** [comment]
Hi @maximilian-mueller67,

We couldn't reproduce the issue with the steps you provided. Looks like we are missing some details that may affect the behavior.
Could you please provide more information about what launches you have and how they originally displayed via ReportPortal UI. What is happening to them on UI when filter applied. And, please, double-check the request that your request doesn't work for the specific launches.
Additional details (screenshots) are welcomed.

Thanks!

**Comment by maximilian-mueller67** [comment]
Hi @alena-babrova I will take a look again and update you

**Comment by maximilian-mueller67** [comment]
Hi @alena-babrova 
I cannot reproduce it anymore; could be that it was fixed in 5.7.4

**Comment by alena-babrova** [comment]
Hi @maximilian-mueller67
Thank you for the update. Good news :)
